src/App.js

```diff
@@ -1,24 +1,52 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import Product from "./components/Product/Product";
+import Search from "./components/Search/Search";
+import Loader from "./utils/loader/loader";
+// import products from "./data/product.js"
 
 function App() {
+  const [products, setProducts] = React.useState(null);
```

Comment:
React.useState, you can import useState as named export, instead of repeating React.useState multiple times

Reply:
Okay Sir. I'll import them in every component needed from now on.